starter-code/css/main.css

```diff
@@ -6,3 +6,158 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+.nav {
+    background-color: white;
+    width: 100vw;
+    height: 60px;
+    top: 12px;
+    position: fixed;
+}
+.nav img {
+    margin-top: -4px;
+    height: 50px;
+    width: 130px;
+    margin-left: 30px;
+}
+
+.nav ul {
+    margin-top: 10px;
+    margin-right: 25px;
+    display: flex;
+    float: right;
+    list-style-type: none;
+}
+
+.nav li {
+    padding-right: 20px;
+}
+
+.top-section {
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-image: url('/Users/Drewskii/Desktop/labs/code/lab-css-spotify-clone/starter-code/images/landing.jpg')
```

Comment:
your path should be `background-image: url('../images/landing.jpg')`. websites can't read a path that is on your pc unless its from your pc. the `../` means go out 1 folder, then you do `images/` to look for the images folder and then you choose the file `landing.jpg`.  if you don't have to leave the current folder your in like in your index html, then you use `./` which is to stay in current folder.